core/src/main/resources/swagger/rawls.yaml

```diff
@@ -2258,6 +2258,12 @@ paths:
           description: Destination workspace must match source's realm, if the source has a realm
           schema:
             $ref: '#/definitions/ErrorReport'
+        '449':
```

Comment:
this status code seemed vague enough to be appropriate here... but apparently this is a non-standard status code. suggestions for a better one? conflict is already used in the case of fatal conflicts, so we need a different code so the UI knows to prompt the user for confirmation